### DISCUSSION TITLE
Single letter virtual folder / path

### DISCUSSION THREAD

**Discussion by D1reW0lf** [requarks/wiki#6758] [Error / Bug Report] [answered]
Hi.

I’m trying to convert an existing wiki to Wiki.js.
The problem here is that I already have a set structure that basically is divided on a A to Z folders / categories.
Since Wiki.js uses that for internal pages I’m unable to use that.
Is there any way to change the internal reserved virtual pages to something else in order to be able to accomplish what I need?
Something like changing the single-character page “a” to “wikijs_a” or something?

Thank you.

**Comment by NGPixel** [comment, marked as answer]
No
